playbooks/setup_forklift.yml

```diff
@@ -34,6 +34,7 @@
         state: 'present'
 
     - name: 'install telemetry dependencies'
+      become: true
```

Comment:
I assume you install these system wide to speed things up on reuse between multiple users.

Reply:
that's correct